_docs_integrate/delete-an-attribute.md

```diff
@@ -0,0 +1,127 @@
+There, the `deletionInfo` of the corresponding own shared Attributes is set with `deletionStatus` `"ToBeDeletedByPeer"` and the `deletionDate` received in the Response.
+
+It is also possible for the peer to [reject the Request]({% link _docs_use-cases/use-case-consumption-reject-incoming-request.md %}) for deleting Attributes as a whole, if they have a valid reason for keeping the Attributes.
+In this case, it is advised to provide a `message` using the [RejectRequestItemParameters]({% link _docs_integrate/data-model-overview.md %}#rejectrequestitemparameters), informing the Sender of the Request about the reasons not to delete the Attributes.
+{: .notice--info}
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/a738fd76-2fe0-4e3e-a0fd-bec86b3b7939" id="ZGDlhHzeKlb-"></iframe></div>
+
+## Delete peer shared Attributes
+
+The actual deletion of a peer shared Attribute happens in a separate step.
+This can either be triggered if the `deletionInfo.deletionDate` is reached, in case the deletion was requested by the owner of the Attribute, or if the peer decides they no longer need the shared Attribute.
+
+To [delete a peer shared Attribute]({% link _docs_use-cases/use-case-consumption-delete-a-peer-shared-attribute-and-notify-peer.md %}) only its `attributeId` must be specified.
+Internally, not just the given Attribute is deleted, but also all its predecessors, given there were any.
+Moreover, if the Attribute had a successor, its `succeeds` property will be set to undefined, as the corresponding predecessor no longer exists.
+Then, a [Notification]({% link _docs_integrate/data-model-overview.md %}#notification) with a [PeerSharedAttributeDeletedByPeerNotificationItem]({% link _docs_integrate/data-model-overview.md %}#peersharedattributedeletedbypeernotificationitem) is generated and sent to the owner of the Attribute, informing them that you deleted the Attribute they shared with you.
+Consequently, the `deletionInfo` of their corresponding own shared Attribute and of all potential predecessors is updated with `deletionStatus` `"DeletedByPeer"` and the time of receiving the Notification as `deletionDate`.
+In case the owner already [deleted their own shared Attribute](#delete-own-shared-attributes), nothing happens.
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/2385af4a-4bfa-43f4-a119-afb51273194d" id="HNEl2zIWWLxc"></iframe></div>
+
+## Delete own shared Attributes
+
+As owner of a shared Attribute, it is always possible to delete your shared Attribute copy.
```

Comment:
- As **the** owner of a shared Attribute, it is always possible to delete your shared Attribute copy.
- I'd also reformulate so that "own shared Attribute" can be used instead of "shared Attribute". Here and in some other places within this paragraph.